corgi/tasks/prod_defs.py

```diff
@@ -346,6 +340,16 @@ def _parse_cpes_from_brew_tags(
         return []
 
 
+def brew_tag_to_et_prefixes(brew_tag: str) -> tuple[str, str]:
+    """Match brew tags in prod_defs with those from ET by reducing them to common prefixes"""
+    trimmed_brew_tag = brew_tag.removesuffix("-candidate")
```

Comment:
I wonder if we should parse these tags into their composite parts eg. we may find ourselves having to munge them in other places.

Reply:
That would be overkill in my opinion.